core/src/main/resources/defaults/blueprints/cdp-opdb-721.bp

```diff
@@ -162,7 +162,14 @@
               "displayName": null,
               "configs": [],
               "base": true
-           }
+           },
+          {
+            "refName": "hbase-HBASERESTSERVER-BASE",
+            "roleType": "HBASERESTSERVER",
```

Comment:
Make sure that CM is setting up HBase REST in a reasonable manner - including requiring SPNEGO on the REST server endpoint. It would be REALLY bad to have it by default be exposed without authentication.

Reply:
Configs in CM look correct to me, e.g. `hbase.rest.ssl.enabled=true` and REST Server TLS/SSL is also enabled.
My test cluster is available if you'd like to take a look: https://console.dps.mow-dev.cloudera.com/cloud/workloads/details/psomogyi-aws-rest/events